libtenzir/builtins/functions/string.cpp

```diff
@@ -140,3 +190,13 @@ TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim", "utf8_trim"})
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim_start",
                                                      "utf8_ltrim"})
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim_end", "utf8_rtrim"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{
+  "capitalize", "utf8_capitalize"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"lower",
+                                                              "utf8_lower"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"reverse",
+                                                              "utf8_reverse"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"title",
+                                                              "utf8_title"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"upper",
+                                                              "utf8_upper"})
```

Comment:
Not sure whether you considered different names, e.g. `lowercase()`, `titlecase()`, etc., but using Python's names can't be too wrong. So feel free to stick ahead with this.

Reply:
Could also use `toupper`, considering we likely want to also have an `isupper` as well.

Reply:
I guess you can still have `upper` and `is_upper`. Python would say `isupper`, although I don't like that.

So it's probably one of those:
- `title` / `is_title`
- `to_title` / `is_title`
- `titlecase` / `is_titlecase` 
- `to_titlecase` / `is_titlecase`

Not that it matters too much for this function (who uses it anyway?), but this will probably establish a pattern we'll use elsewhere.

Reply:
Yeah, not sure about that one anyways, I mostly ported it because it was essentially free. 

I think I like `is_x` and `to_x` best. Might drop `to_title` for now until we have a need for it.

Reply:
I settled on `to_upper`, `to_title`, and `to_lower` for now after giving this some more thought. If you agree, please resolve this.